src/contracts/libraries/SlashingLib.sol

```diff
@@ -23,13 +23,11 @@ uint64 constant WAD = 1e18;
  * Note that `withdrawal.scaledShares` is scaled for the beaconChainETHStrategy to divide by the beaconChainScalingFactor upon queueing
  * and multiply by the beaconChainScalingFactor upon withdrawal
  */
-struct StakerScalingFactors {
-    uint184 depositScalingFactor;
-    uint64 beaconChainScalingFactor;
-    bool isBeaconChainScalingFactorSet;
+struct DepositScalingFactor {
```

Comment:
Could probably remove the struct

Reply:
Unless the clarity with the lib functions is preferred which I assume is why it was left in

Reply:
Yeah this felt like a clarity thing, but I'm not sold one way or another. I may reconsider as I finish this refactor.

Reply:
is custom type the move?